includes/class-kadence-blocks-prebuilt-library-rest-api.php

```diff
@@ -636,6 +636,19 @@ public function handle_event( $request ) {
 					'collections'       => $event_data['customCollections'] ?? '',
 				];
 				break;
+			case 'ai_wizard_complete':
+				$event = 'AI Wizard Complete';
+				$context = [
```

Comment:
Do we want to `array_filter` to remove keys with empty values, or do we want to submit blank values? 

I'm not sure how Mixpanel filters/groups things, but if grouping by `organization_type === ''` may make for some funky reports.